### DISCUSSION TITLE
24 hour forecasts API

### DISCUSSION THREAD

**Discussion by jgoldverg** [electricitymaps/electricitymaps-contrib#7644] [General]
Hello All,

I am a PhD student and was hoping if I could get access to the forecast api for a publication I am working on. I am looking for the US region specifically but would be open to including other countries (India, Canada).

Thank you,
Jacob

**Comment by VIKTORVAV99** [comment]
I would ask you to fill out the form here: https://www.electricitymaps.com/research

And then the right people can get back to you!
